Training/Program.cs

```diff
@@ -25,25 +23,25 @@ internal class Program {
             else if (inputStr.Length == 1)
                Console.WriteLine ($"And the winner is: {inputStr.ToUpper ()} with {inputStr.Length} vote\nPress '~' and 'enter' to Quit");
             else if (Regex.IsMatch (inputStr, "^[a-zA-Z]+$")) {
-               string winner = VotingContest (ref inputStr);
-               Console.WriteLine ($"And the winner is: {winner.FirstOrDefault ()} with {winner.Length} votes");
+               (string, int) winner = VotingContest (inputStr);
+               Console.WriteLine ($"And the winner is: {winner.Item1.First ()} with {winner.Item2} votes");
                Console.WriteLine ("Press '~' and 'enter' to Quit");
                continue;
             } else
                Console.WriteLine ("Enter a valid string");
          }
       }
       /// <summary></summary>
-      /// <param name="str">User input</param>
+      /// <param name="inputStr">User input</param>
       /// <returns>Winner of the voting contest</returns>
-      static string VotingContest (ref string str) {
-         char[] contestents = str.ToUpper ().ToCharArray ();
+      static (string, int) VotingContest (string inputStr) {
+         char[] contestents = inputStr.ToUpper ().ToCharArray ();
          Array.Sort (contestents);
          string result = "";
          for (int i = 0; i < contestents.Length - 1; i++) result += contestents[i] == contestents[i + 1] ? contestents[i] : contestents[i] + ",";
-         return result.Split (',').OrderByDescending (g => g.Length).FirstOrDefault ();
+         return (result.Split (',').OrderByDescending (g => g.Length).First (), result.Split (',').OrderByDescending (g => g.Length).First ().Length);

```

Comment:
Two times, You are doing Split and OrderByDescending. Call it only one time and return the winning character and the count.
` static (char, int) VotingContest (string inputStr) {`
.
.
.
.
```
var res = result.Split (',').OrderByDescending (x => x.Length).First ();
return (res.First (), res.Length);
}
```